README.md

```diff
@@ -2,31 +2,1241 @@
+      returns a `double`.
+   5. Add the following code to override the `toString()` method:
+
+```java
+    @Override
+    public String toString() {
+        return String.format("Ticket Number = %d, Price = %.2f", ticketNumber, getPrice());
+    }
+```
+
+This task can be completed now.
+
+## Task 2 - Evolve the Array to a List
+
+Currently, the `ConcertRepository` class has an array of concerts. But now that
+we have learned about more data structures, let's replace the array with a
+`List`. Since a `List` is dynamically sized, as in we can continue to add
+elements to it, we won't be restricted by how many concerts we could have!
+
+1. Modify the `ConcertRepository` class.
+   1. Replace the array of `Concert` objects with a `List` of `Concert` objects.
+   2. Make the appropriate adjustments.
+   3. Remove the instance variable `currentSize`.
+   4. Remove the method `getCurrentSize()`.
+   5. Create a method called `getAllConcerts()` that takes no parameters and
+      returns the `List` of `Concert` objects.
+2. Modify the `ConcertService` appropriately given the changes made to the
+   `ConcertRepository` class.
+3. Modify the following unit tests below and ensure the tests all pass.
+
+Update the `ConcertRepositoryTest`:
+
+```java
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ConcertRepositoryTest {
+
+    private ConcertRepository repository;
+
+    @BeforeEach
+    void setUp() {
+        repository = new ConcertRepository();
+    }
+
+    @Test
+    void constructor() {
+
+        // current size is 0 since no concerts have been added
+        assertEquals(0, repository.getAllConcerts().size());
+    }
+
+    @Test
+    void addGet1() {
+        assertEquals(0, repository.getAllConcerts().size());
+
+        // add a concert
+        assertTrue(repository.add(new Concert("Artist0", 1000)));
+        assertEquals(1, repository.getAllConcerts().size());
+
+        // retrieve the concert using index 0
+        Concert c = repository.get(0);
+        assertEquals("Artist0", c.getPerformer());
+        assertEquals(1000, c.getAvailable());
+
+    }
+
+    @Test
+    void addGet5() {
+        assertEquals(0, repository.getAllConcerts().size());
+
+        // add 5 concerts
+        for (int i = 0; i < 5; i++) {
+            // add the concert
+            assertTrue(repository.add(new Concert("Artist" + i, i * 1000)));
+
+            // retrieve the concert using index i
+            assertNotNull(repository.get(i));
+
+            // confirm the current size
+            assertEquals(i + 1, repository.getAllConcerts().size());
+        }
+
+        // confirm the size did not increase
+        assertEquals(5, repository.getAllConcerts().size());
+    }
+
+    @Test
+    void getConcertState() {
+        // add 3 concerts
+        assertTrue(repository.add(new Concert("The Weekend", 1000)));
```

Comment:
Really Very Nit: 'The Weeknd' (without the final e)

Reply:
This looks fixed on my end. Is it fixed on yours, Jay?

Reply:
Yep, all good now